react/week2/todoapp/src/todo-app.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import Timer from './Timer'
+
+function TodosList({ id, description, removeTodo }) {
+    const [isDone, setIsDone] = useState('');
+    return (
+        <div>
+            <li className="list-of-todos" style={isDone ? { textDecoration: "line-through" } : { textDecoration: "" }}>{description}
+                <span>
+                    <input type="checkbox" onClick={() => setIsDone(!isDone)} />
+                    <button onClick={() => removeTodo(id)}>Delete</button>
+                </span>
+            </li>
+        </div>
+    )
+}
+
+const RenderedTodos = ({ todos }) => {
+    const [addTodo, setAddTodo] = useState(todos);
+    const [value, setValue] = useState('');
+    //adding the todos to todos array
+    const OnClickAddTodo = () => {
+        if (value !== "") {
+            const myNextTodos = { id: addTodo.length + 1, description: value };
+            const myNextTodoNames = addTodo.concat(myNextTodos);
+            setAddTodo(myNextTodoNames)
```

Comment:
We do not want to see in the input after adding in list . Right? so what we can do is we can add 
setValue(''); 
before setAddTodo(myNextTodoNames). Try can see the difference and understand what is happening.